Lib/configparser.py

```diff
@@ -687,7 +688,7 @@ def read(self, filenames, encoding=None):
 
         Return list of successfully read files.
         """
-        if isinstance(filenames, str):
+        if isinstance(filenames, (str, bytes, os.PathLike)):
```

Comment:
Thanks! I've added the module maintainer to that issue.

Actually I want to get this PR in for 3.6.1 which is due this weekend :)

With or without pathlib support, accepting bytes objects is a new feature and it cannot be added to bugfix releases.